tests/api/test_generated_endpoints.py

```diff
@@ -94,6 +95,45 @@ def test_get_data(self, generator, model, route, controlled_values, client, Grap
         for value in controlled_values:
             assert actual['data']['attributes'][value] == expected[camelCase_to_underscore(value)]
 
+    def test_can_delete_work(self, client, normalized_data_id):
+        graph = ChangeGraph([{
```

Comment:
Rather than building up the entire change graph, can you just call `factories.AbstractCreativeWork(is_deleted=True)` to get a deleted work.